Update README.md for unit tests (#32758)
joomla/joomla-cms: tests/Unit/README.md

1. Checkout the current Joomla 4.0 development branch from Github. (https://github.com/joomla/joomla-cms.git Branch `4.0-dev`)
2. Run `composer install` in the root of your checkout.

> 3. Adjust the file `/joomla-cms/phpunit-pgsql.xml.dist` und/oder `joomla-cms/phpunit-pgsql.xml.dist` based on your environment.

Reviewer: In english please ;) I am guessing it should be 
```suggestion
3. Edit the file `/joomla-cms/phpunit-pgsql.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.
```

Author: Thanks

> 3. Edit the file `/joomla-cms/phpunit-pgsql.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.

Reviewer 2: ```suggestion
3. Edit the file `joomla-cms/phpunit.xml.dist` and/or `joomla-cms/phpunit-pgsql.xml.dist` depending on your environment.
```
